File: app/src/main/java/com/rdev/tryp/blocks/reward_profile/RewardPointsFragment.java
package com.rdev.tryp.blocks.reward_profile;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;
import android.widget.Toast;

import com.rdev.tryp.ContentActivity;
import com.rdev.tryp.R;

import afu.org.checkerframework.checker.nullness.qual.NonNull;
import afu.org.checkerframework.checker.nullness.qual.Nullable;
import androidx.fragment.app.Fragment;

public class RewardPointsFragment extends Fragment implements View.OnClickListener {
    private View root;

    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        root = inflater.inflate(R.layout.fragment_reward, container, false);

        ImageView fab = root.findViewById(R.id.back_btn);
        fab.setOnClickListener(this);
        root.findViewById(R.id.btnWithdraw).setOnClickListener(this);

        return root;
    }

    @Override
    public void onClick(View v) {
        switch (v.getId()){
            case R.id.back_btn:
                ((ContentActivity) getActivity()).startFragment(ContentActivity.TYPE_REWARDS);
                break;
            case R.id.btnWithdraw:
                ((ContentActivity) getActivity()).startFragment(ContentActivity.TYPE_REWARDS_WITHDRAW);
                break;
        }
    }
}

File: app/src/main/java/com/rdev/tryp/blocks/reward_profile/RewardPointsFragment.kt
package com.rdev.tryp.blocks.reward_profile

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ImageView

import com.rdev.tryp.ContentActivity
import com.rdev.tryp.R

import androidx.fragment.app.Fragment

class RewardPointsFragment : Fragment(), View.OnClickListener {
    private var root: View? = null

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        root = inflater.inflate(R.layout.fragment_reward, container, false)

        val fab = root?.findViewById<ImageView>(R.id.back_btn)
        fab?.setOnClickListener(this)
        root?.findViewById<View>(R.id.btnWithdraw)?.setOnClickListener(this)

        return root
    }

    override fun onClick(v: View) {
        when (v.id) {
            R.id.back_btn -> (activity as ContentActivity).startFragment(ContentActivity.TYPE_REWARDS)
            R.id.btnWithdraw -> (activity as ContentActivity).startFragment(ContentActivity.TYPE_REWARDS_WITHDRAW)
        }
    }

}